core/CMake_symbol_check.cmake

```diff
@@ -38,6 +38,25 @@
 
 include(${lib_fileloc}.cmake)
 
+# XXX i#3450: Return whether binutils is version 2.30.
```

Comment:
There is no missing work here, right?  No reason for an "XXX"?  s/XXX//.

Reply:
Changed to xref.